AllReadyApp/Web-App/AllReady/Features/Resource/ResourcesByCategoryQueryHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using AllReady.Models;
+using MediatR;
+
+namespace AllReady.Features.Resource
+{
+    public class ResourcesByCategoryQueryHandler : IRequestHandler<ResourcesByCategoryQuery, List<Models.Resource>>
+    {
+        private readonly IAllReadyDataAccess dataAccess;
+
+        public ResourcesByCategoryQueryHandler(IAllReadyDataAccess dataAccess)
+        {
+            this.dataAccess = dataAccess;
+        }
+
+        public List<Models.Resource> Handle(ResourcesByCategoryQuery message)
+        {
+            return dataAccess.GetResourcesByCategory(message.Category).Distinct().ToList();
```

Comment:
Practically, it doesn't matter. `Resources` aren't used for anything currently and I don't know if we even have a clear story for what they mean or what purpose they serve. They were somewhat hastily thrown in the code base as part of a codeathon a long time ago and haven't been touched since.

I say take it away and keep the code clean or else others will think it's there for a reason.